### PR TITLE
Checkout tutorial: multi-step

### PR DESCRIPTION
**Pull request by jeff-matthews**
This pull request (PR) adds a new tutorial for the checkout drop-in component. It's based on the docs that the engineering team created in the checkout repo for the GA demo.

I removed the "set up" and "initialize" steps (https://github.com/commerce-docs/microsite-commerce-storefront/pull/130/commits/b9deab8c5cce1625c4b3f33d9d4ef5aaa921bc2d) that were included in the original demo because that content already exists in the latest `commerce-checkout.js` block file in the boilerplate (`develop` branch).

## Staging

https://commerce-docs.github.io/microsite-commerce-storefront/dropins/checkout/tutorials/multi-step/

### CODE REVIEW

File: src/content/docs/dropins/checkout/tutorials/multi-step.mdx [hunk 1]
```diff
@@ -0,0 +1,1404 @@
...
+   ![Empty cart](@images/dropins/checkout/empty-cart.png)
+   </Diagram>
+</Steps>
+</Task>
+
+<Task>
+### Handle shipping method form
+
+<Steps>
+1. You've probably noticed that even after initializing the checkout the shipping method form still loads. To resolve this, create a new utility method called `continueToShipping` to remove the skeleton and render the shipping method form when the checkout is initialized.
```
Comment (jaumecapdevila) [inline src/content/docs/dropins/checkout/tutorials/multi-step.mdx]: I think this is not correct, the form we are rendering here is the shipping address form 👍🏻

File: src/content/docs/dropins/checkout/tutorials/multi-step.mdx [hunk 2]
```diff
@@ -0,0 +1,1404 @@
...
+
+<Task>
+### Create order confirmation
+
+The last step to complete the multi-step checkout process is to create an order confirmation page. To make things easy, you can reuse the code from the `commerce-checkout.js` block in the boilerplate template.
+
+<Steps>
+1. Copy the `displayOrderConfirmation` function.
+
+1. Copy the `handleCheckoutOrder` handler.
```
Comment (jaumecapdevila) [inline src/content/docs/dropins/checkout/tutorials/multi-step.mdx:718]: Should we update the steps order or just use a non ordered list?

**Review by jaumecapdevila — APPROVED** [review]

File: src/content/docs/dropins/checkout/tutorials/multi-step.mdx (hunk 1 above)
Comment (jeff-matthews) [reply src/content/docs/dropins/checkout/tutorials/multi-step.mdx]: Good catch!

File: src/content/docs/dropins/checkout/tutorials/multi-step.mdx (hunk 2 above)
Comment (jeff-matthews) [reply src/content/docs/dropins/checkout/tutorials/multi-step.mdx:718]: Actually, our static site generator takes care of the numbering. We always specify "1" for maintainability. This is how it looks after the site is built:

Screenshot 2024-12-11 at 9 55 38 AM

File: src/content/docs/dropins/checkout/tutorials/multi-step.mdx [hunk 3]
```diff
@@ -0,0 +1,1404 @@
+---
+title: Multi-step checkout
```
Comment (keharper) [inline src/content/docs/dropins/checkout/tutorials/multi-step.mdx]: Consider adding "guest" to the title to make the scope of the tutorial immediately clear.

File: src/content/docs/dropins/checkout/tutorials/multi-step.mdx [hunk 4]
```diff
@@ -0,0 +1,1404 @@
...
+  }
+
+  .checkout__aside {
+    order: -1;
+  }
+}
+```
+
+<Diagram caption="Render containers with styles">
+![Render containers with styles](@images/dropins/checkout/render-containers-no-styles.png)
```
Comment (keharper) [inline src/content/docs/dropins/checkout/tutorials/multi-step.mdx]: I was looking at my local build, trying to figure out what the CSS changed. You're pointing to the "before" screenshot. 😆

**Review by keharper — APPROVED** [review]

**Pull request merged**